src/OrbitTest/OrbitTestImpl.cpp

```diff
@@ -95,15 +95,15 @@ void NO_INLINE OrbitTestImpl::BusyWork(uint64_t microseconds) {
   }
 }
 
-#if ORBIT_API_ENABLED
 static void NO_INLINE SleepFor1Ms() { std::this_thread::sleep_for(std::chrono::milliseconds(1)); }
 
 static void NO_INLINE SleepFor2Ms() {
   ORBIT_SCOPE("Sleep for two milliseconds");
   ORBIT_SCOPE_WITH_COLOR("Sleep for two milliseconds", kOrbitColorTeal);
   ORBIT_SCOPE_WITH_COLOR("Sleep for two milliseconds", kOrbitColorOrange);
   thread_local uint64_t group_id = 0;
-  uint64_t current_id = group_id++;
+  // [[maybe_unused]] prevents "unused variable" compilation error if ORBIT_API_ENABLED is set to 0.
+  [[maybe_unused]] uint64_t current_id = group_id++;
```

Comment:
nit. Not compiling it altogether might be cleaner. Maybe use `#ifdef ORBIT_API_ENABLED`, or possibly introduce a macro, something like `IF_ORBIT_API_ENABLED(X)` that would be defined to nothing when `ORBIT_API_ENABLED` is 0. Wdyt?

Reply:
I find `[[maybe_unused]]` cleaner than using `#ifdef`s or new macros because the only problem really is that we enforce variables to be used&mdash;we don't want client code to have to use `ORBIT_API_ENABLED` directly or indirectly.